tests/testthat/test_build_glm.R

```diff
@@ -118,12 +118,25 @@ test_that("prediction with categorical columns", {
   model_data <- build_glm(test_data, family = "binomial", CANCELLED ~ `Carrier Name` + CARRIER + DISTANCE, test_rate = 0.6)
 
   ret <- prediction(model_data, test_data, type.predict = "response")
+  both_ret <- prediction(model_data, test_data)
+  train_ret <- prediction(model_data, test_data, test = FALSE)
+
   expect_true(nrow(ret) > 0)
   expect_true(all(ret["Fitted"] >= 0 & ret["Fitted"] <= 1))
   expect_equal(colnames(ret), c("CANCELLED", "Carrier.Name", "CARRIER", "DISTANCE", "Fitted", "Standard Error"))
 
+  expect_true(all(both_ret["Fitted.response"] >= 0 & both_ret["Fitted.response"] <= 1))
```

Comment:
Fitted.response appears in model info view but fitted.response appears in data frame, so I think lower case is better for it